### PR TITLE
CASSANDRA-15835 Update Docker image w/ JAVA8_HOME via ENV

### PR DESCRIPTION
**Pull request by snazy**

### CODE REVIEW

**Review by michaelsembwever — APPROVED** [review]
👍

**Review by nastra — APPROVED** [review]
LGTM. I'll publish a new version of the ubuntu1910 images once this is merged

File: docker/testing/ubuntu18_j11.docker [hunk 1]
```diff
@@ -80,6 +80,11 @@ RUN echo 'export ANT_HOME=/usr/share/ant' >> /home/cassandra/.bashrc && \
     echo 'export JAVA11_HOME=/usr/lib/jvm/java-11-openjdk-amd64' >> /home/cassandra/.bashrc && \
     echo 'export JAVA_HOME=$JAVA8_HOME' >> /home/cassandra/.bashrc
 
+ENV ANT_HOME=/usr/share/ant \
```
Comment (nastra) [inline docker/testing/ubuntu18_j11.docker:83]: you can't really build this docker image anymore, so not sure we actually need these changes here

File: docker/testing/ubuntu1810_j11.docker [hunk 2]
```diff
@@ -80,6 +80,11 @@ RUN echo 'export ANT_HOME=/usr/share/ant' >> /home/cassandra/.bashrc && \
     echo 'export JAVA11_HOME=/usr/lib/jvm/java-11-openjdk-amd64' >> /home/cassandra/.bashrc && \
     echo 'export JAVA_HOME=$JAVA8_HOME' >> /home/cassandra/.bashrc
 
+ENV ANT_HOME=/usr/share/ant \
```
Comment (nastra) [inline docker/testing/ubuntu1810_j11.docker:83]: you can't really build this docker image anymore, so not sure we actually need these changes here

**Comment by nastra** [comment]
I published `nastra/cassandra-testing-ubuntu1910-java11:20200603` and `nastra/cassandra-testing-ubuntu1910-java11-w-dependencies:20200603` with your changes @snazy

**Comment by michaelsembwever** [comment]
> I published nastra/cassandra-testing-ubuntu1910-java11:20200603 and nastra/cassandra-testing-ubuntu1910-java11-w-dependencies:20200603 with your changes @snazy

Can we follow up (separately) how we go about deploying these images (and to where).

I'm thinking that cassandra-builds should have its "Building packages" instructions put into a build script and put into ASF CI, and investigate if we can CD those builds into dockerhub…

**Pull request merged**